src/default_client.rs

```diff
@@ -922,9 +1673,29 @@ impl<A: ClientAdapter + Sync + Send> Client for DefaultClient<A> {
         self.send(request).await
     }
 
+    /// List current or upcoming
+    ///
+    /// TODO: Document all
```

Comment:
The official docs https://heroiclabs.com/docs/gameplay-tournaments/#list-tournament-records don't mention the different flags. E.g. what are `category_start` and `category_end`?

Reply:
@marot that's the link for tournament records. Take a look here: https://heroiclabs.com/docs/gameplay-tournaments/#list-tournaments